.yarnrc.yml

```diff
@@ -0,0 +1,314 @@
+compressionLevel: 0
+
+enableGlobalCache: true
+
+logFilters:
+  - level: discard
+    text: "@automattic/composite-checkout@workspace:packages/composite-checkout provides webpack (pa6811) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "@automattic/composite-checkout@workspace:packages/composite-checkout [abd57] provides webpack (pb8614) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "@automattic/notifications@workspace:apps/notifications provides webpack (pea42a) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "@automattic/o2-blocks@workspace:apps/o2-blocks provides webpack (p0f649) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "@automattic/wpcom-block-editor@workspace:apps/wpcom-block-editor provides webpack (p2325f) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "@automattic/wpcom-editing-toolkit@workspace:apps/editing-toolkit provides webpack (pa616b) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "calypso@workspace:client provides webpack (pab2bd) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - level: discard
+    text: "wp-calypso@workspace:. provides webpack (p39759) with version 5.46.0, which doesn't satisfy what @automattic/calypso-build and some of its descendants request"
+  - code: YN0013
+    level: discard
+
+nodeLinker: node-modules
+
+packageExtensions:
```

Comment:
Not auto generated, I had to compile it by hand. Eventually it will disappear as yarn prints a warning when you run `yarn` and a `packageExtension` doesn't match any package (assuming we keep an eye on those warnings, of course)

Reply:
> assuming we keep an eye on those warnings, of course

You can change them to be errors with `logFilters` if you want